lib/node_modules/@stdlib/math/base/special/bernoulli/src/main.c

```diff
@@ -173,6 +173,9 @@ double stdlib_base_bernoulli( const int32_t n ) {
 	if ( stdlib_base_is_nan( n ) || !stdlib_base_is_nonnegative_integer( n ) ) {
```

Comment:
@gunjjoshi We should rethink these checks. 1. `n` can never be `NaN`. 2. `stdlib_base_is_nonnegative_integer` assumes a `double` as an input argument. Here we can just use `n < 0`.

Reply:
Yes, that does make sense. Also, we have already excluded the test in which `NaN` is given as the input for `test.native.js`, while we have that in `test.js`. I have made these changes, thanks!